pkg/gadgets/helpers.go

```diff
@@ -16,10 +16,20 @@ package gadgets
 
```

Comment:
IMO all additions to this file should be moved to a new `networkgadgets.go` file as they don't have a direct relation with the helpers implemented here.

Reply:
Yep, I think we should move to a separate file. What about creating a package in pkg/gadgets/internal/networktracer?

Reply:
Doing so, we can do the following within the DNS and SNI tracers:

```golang
type Tracer struct {
	*networktracer.Tracer[types.Event]
}
```